setup/Makefile

```
@@ -8,7 +8,7 @@ build-neutron:
 		@docker buildx build --load --build-context app=$(APP_DIR)/neutron -t neutron-node -f dockerbuilds/Dockerfile.neutron --build-arg BINARY=neutrond .
 
 build-hermes:
-		@docker build -f dockerbuilds/Dockerfile.hermes -t hermes:1.0.0 .
+		@docker build -f dockerbuilds/Dockerfile.hermes -t hermes:a285c01 .
```

I think this little change will give us better understanding of which hermes version we currently are at.
```suggestion
		@docker build -f dockerbuilds/Dockerfile.hermes -t hermes:1.3.0-a285c01 .
```